kafka-ballerina/listener.bal

```diff
@@ -23,12 +23,15 @@ public client class Listener {
     public ConsumerConfiguration consumerConfig;
     private string keyDeserializerType;
     private string valueDeserializerType;
+    private string bootstrapServers;
 
     # Creates a new Kafka `Listener`.
     #
+    # + bootstrapServers - List of remote server endpoints of kafka brokers
     # + config - Configurations related to consumer endpoint
     # + return - A `kafka:Error` if an error is encountered or else '()'
-    public isolated function init (ConsumerConfiguration config) returns Error? {
+    public isolated function init (string bootstrapServers, *ConsumerConfiguration config) returns Error? {
```

Comment:
string|string[] bootstrapServers

Reply:
fixed in 21905ae05f4fe648652dd2d906696c47ee0d4b4e